protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -522,6 +524,41 @@ message CommitMultipartVersionedBlobArtifact {
     }
 }
 
+message AddRepositoryAttributes {
+    RepositoryIdentification repository_id = 1;
+    repeated common.KeyValue attributes = 2;
+
+    message Response {
+    }
+}
+
+message UpdateRepositoryAttribute {
+    RepositoryIdentification repository_id = 1;
+    common.KeyValue attribute = 2;
+
+    message Response {
+    }
+}
+
+message GetRepositoryAttributes {
```

Comment:
not required

Reply:
Removed `getRepositoryAttributes` endpoint